homework-12/ToDoList/main.js

```diff
@@ -0,0 +1,39 @@
+const addTask = document.querySelector("#addtask");
+const taskTitle = document.querySelector("#taskTitle");
+const uList = document.querySelector("#ulList");
+
+addTask.addEventListener("click", (e) => {
+  if (taskTitle.value === "") {
+    alert("Please enter something");

```

Comment:
Let's move from `alert` to something useful, as an option showing an error message below the input

Also, it's nice to have RegExp  or any other validation for empty lines

https://user-images.githubusercontent.com/84010501/199434406-dbc18af8-3fb9-4341-aa3b-68f468848843.mov